### ISSUE TITLE
TECH TASK: affinityworks github url [1]

### ISSUE THREAD

**Issue by rabble** [affinityworks/main#463]
## Acceptance Criteria

* navigating to https://github.com/affinityworks/affinityworks should go to this repo
* opening a PR against master at that URL should still trigger the deployment of a review app
* merging to master at this new URL should still trigger heroku to promote a commit to the dev server
* doing the above should still cause messages to appear in the slack `#code` channel

## Blockers

* [x] cannot toggle CI tests on for  re-connected heroku pipeline (need owner permissions)
* [x] cannot rename heroku apps `advocacycommons` -> `affinityworks` (need owner permissions)
* [ ] we probably want to rename slack domain to `affinityworks.slack.com`, but i do not see any available UI elements to allow me to do that (likely because i lack permissions)

**Comment by aguestuser** [comment]
+1

also: perhaps our github page should also be linked to from our splash site? (by "splash site" i mean  https://affinity.works) i remember looking for this repo from there and being confused i couldn't find it.)

**Comment by aguestuser** [comment]
@rabble @Matt-Holland see blockers above.

**Comment by aguestuser** [comment]
![heroku-ci](https://user-images.githubusercontent.com/6032844/35006451-b2767d9c-fac4-11e7-863b-1a56b1fd2f44.png)

**Comment by aguestuser** [comment]
![heroku-edit-name](https://user-images.githubusercontent.com/6032844/35006461-b9218f06-fac4-11e7-963f-6be1088fd9fa.png)

**Comment by Matt-Holland** [comment]
NOTE: We are stuck trying to transfer ownership of staging and prod to the advocacycommons heroku team, matt has a ticket open with Heroku to help.

**Comment by Matt-Holland** [comment]
This is dependent on #478

**Comment by aguestuser** [comment]
closing, as all acceptance criteria are met. renaming slack is out of scope for this card. CI tests do not run on open PR, but that is a separate card: #481
